SPEC.md

```diff
@@ -83,10 +85,18 @@ Success is indicated by a return code of zero and the following JSON printed to
     "ip": <ipv6-and-subnet-in-CIDR>,
     "gateway": <ipv6-of-the-gateway>,  (optional)
     "routes": <list-of-ipv6-routes>    (optional)
-  }
+  },
+  "dns": <list-of-DNS-nameservers>     (optional)
 }
 ```
 
+The "dns" field contains a list of priority ordered list of DNS nameservers that this network is aware of.
+Each entry in the list is a string containing either an IPv4 or an IPv6 address.
+Typically this value would just be the value returned by the IPAM plugin.
+It is outside the scope of this specification on how the container runtime uses the list of DNS nameservers from each of the networks to create a final list for the container.
+In addition, it is outside the scope of this specification how the final list is used to provide name resolution services to the container.
```

Comment:
> doesn't this imply line-96 anyway? maybe we could just have one

Yeah, how about:

"It is outside the scope of this specification how the container runtime uses the list of DNS nameservers from each of the networks to provide name resolution services to the container."